### ISSUE TITLE
"Create new sketch" feature 

### ISSUE THREAD

**Issue by funwithtriangles** [nudibranchrecords/hedron#465]
Give it a name and a description and it creates a sketch and config file for you, with a very simple spinning cube with a single param and shot as a starter. Faster than writing yourself or copy pasting.

Could eventually have other options to help you get started (e.g. add transform controls / is a postprocessing effect)

**Comment by cale-bradbury** [comment]
could be good if this could also manage the global.d.ts file with hedron deps (maybe renaming to hedron.d.ts to help make it clear that folks should not add to it)

**Comment by funwithtriangles** [comment]
Yeah definitely, although I think we can even have this generated when you choose your sketches dir. Maybe the sketches server checks to see if you're using TS, if so, a prompt appears explaining and asks to generate the file.
